### PR TITLE
fix icon for course page "explanation"

### LINKED ISSUE #325: course page icon "explanation" is broken

### PR DESCRIPTION
**Pull request by Knorrke**
fixes #325

### CODE REVIEW

**Comment by arekkas** [comment]
why not "explanation" ?

File: src/module/Entity/src/Entity/Form/ModulePageForm.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ function __construct(LicenseInterface $license)
         $select->setLabel('Select an icon');
         $select->setAttribute('id', 'icon');
         $select->setValueOptions(array(
-                '' => 'Explanation',
+                'none' => 'Explanation',
```
Comment (arekkas) [inline src/module/Entity/src/Entity/Form/ModulePageForm.php]: why not "explanation" ?

Comment (Knorrke) [inline src/module/Entity/src/Entity/Form/ModulePageForm.php]: Would be possible too, should I change it? In frontend this key is used for selecting the fontawesome icon, so the names of those icons is used here. But explanation should not have an icon, hence "none". But "explanation" would work fine too.

Comment (arekkas) [inline src/module/Entity/src/Entity/Form/ModulePageForm.php]: yes change it :)

**Pull request merged**